README.md

```diff
@@ -4,59 +4,22 @@
 
 This repository is used to generate the [documentation website for the Tech Docs Template][tdt-docs] and uses the template itself.
 
-The Tech Docs Template is a [middleman template][mmt] that
-you can use to build technical documentation websites using a GOV.UK style.
+The Tech Docs Template is a [middleman template][mmt] that you can use to build technical documentation websites using a GOV.UK style.
 
-## Before you start
-
-To use the Tech Docs Template you need:
-
-- [Ruby][install-ruby]
-- [Middleman][install-middleman]
-
-## Making changes
-
-To make changes to the documentation for the Tech Docs Template website, edit files in the `source` folder of this repository.
-
-You can add content by editing the `.html.md.erb` files. These files support content in:
-
-- Markdown
-- HTML
-- Ruby
-
-👉 You can use Markdown and HTML to [generate different content types][example-content] and [Ruby partials to manage content][partials].
-
-👉 Learn more about [producing more complex page structures][multipage] for your website.
-
-## Preview your changes locally
-
-To preview your new website locally, navigate to your project folder and run:
-
-```sh
-bundle exec middleman server
-```
-
-👉 See the generated website on `http://localhost:4567` in your browser. Any content changes you make to your website will be updated in real time.
-
-To shut down the Middleman instance running on your machine, use `ctrl+C`.
-
-If you make changes to the `config/tech-docs.yml` configuration file, you need to restart Middleman to see the changes.
+To find out more about setting up and managing content for a website using this template, see the [Tech Docs Template documentation](https://tdt-documentation.london.cloudapps.digital/).
```

Comment:
Also wonder whether this is needed? We have the link to the documentation in the first paragraph so we've probably said all we need to up there.